pkg/meta/base.go

```diff
@@ -708,21 +710,41 @@ func (m *baseMeta) flushQuotas() {
 			}
 		}
 		m.quotaMu.RUnlock()
-		// FIXME: merge
-		for ino, q := range quotas {
-			if err := m.en.doFlushQuota(Background, ino, q.newSpace, q.newInodes); err != nil {
-				logger.Warnf("Flush quota of inode %d: %s", ino, err)
+
+		for ino := range quotas {
+			inodes = append(inodes, ino)
+		}
+		batch := 1000
+		if m.en.enType() == "tkv" {
```

Comment:
the number of active quota should be large, and updating 100 should not be much slower than updating 20, since all the quotas should belong to single TiKV region, and we can get/put them in single batch.

That means we don't need the batch here.